doc/examples/color_exposure/plot_adapt_hist_eq_3d.py

```diff
@@ -0,0 +1,246 @@
+                  for j in [0, 1] for k in [0, 1]]).astype(np.float32)
+lines = [np.array([i, j]) for i in verts
+         for j in verts if np.allclose(np.linalg.norm(i - j), 1)]
+
+# "render" volumetric data
+for iax, ax in enumerate(axs[:]):
+    plt_render_volume(ims[iax], ax, cmap, 0, 1, [2, 2, 2], 20)
+
+    # plot 3D box
+    rect_shape = np.array(im_orig.shape) + 2
+    for line in lines:
+        ax.plot((line * rect_shape)[:, 0] - 1,
+                (line * rect_shape)[:, 1] - 1,
+                (line * rect_shape)[:, 2] - 1,
+                linewidth=1, color='grey')
+
+# add boxes illustrating the kernels
+ns = np.array(im_orig.shape) // kernel_size - 1
+for iax, ind, rect_shape in zip([1] + [2] * 4,
+                                [[0, 0, 0],
+                                 [ns[0] - 1, ns[1], ns[2] - 1],
+                                 [ns[0], ns[1] - 1, ns[2] - 1],
+                                 [ns[0], ns[1], ns[2] - 1],
+                                 [ns[0], ns[1], ns[2]]],
+                                [np.array(im_orig.shape)]
+                                + [kernel_size] * 4):
+
+    for line in lines:
+        axs[iax].plot(((line + ind) * rect_shape)[:, 0],
+                      ((line + ind) * rect_shape)[:, 1],
+                      ((line + ind) * rect_shape)[:, 2],
+                      linewidth=1.2, color='crimson')
+
+# plot degradation function
+axs[3].scatter(xs=np.arange(len(sigmoid)),
+               ys=np.zeros(len(sigmoid)) + im_orig.shape[1],
+               zs=sigmoid * im_orig.shape[2],
+               s=5,
+               c=get_rgba(sigmoid,
+                          cmap=cmap, vmin=0, vmax=1, alpha=1.)[:, :3])
+
+# subplot aesthetics
+for iax, ax in enumerate(axs[:]):
+
+    # Get rid of panes and axis lines
+    for dim_ax in [ax.w_xaxis, ax.w_yaxis, ax.w_zaxis]:
+        dim_ax.set_pane_color((1., 1., 1., 0.))
+        dim_ax.line.set_color((1., 1., 1., 0.))
+
+    # Make axes limits, see https://github.com/
+    # matplotlib/matplotlib/issues/17172#issuecomment-617546105
+    xyzlim = np.array([ax.get_xlim3d(),
+                       ax.get_ylim3d(),
+                       ax.get_zlim3d()]).T
+    XYZlim = np.asarray([min(xyzlim[0]), max(xyzlim[1])])
+    ax.set_xlim3d(XYZlim)
+    ax.set_ylim3d(XYZlim)
+    ax.set_zlim3d(XYZlim * 3 / 4)
+
+    ax.set_xlabel('x', labelpad=-25)
+    ax.set_ylabel('y', labelpad=-25)
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.set_zticks([im_orig.shape[2] * 2 / 3.])
+    ax.set_zticklabels('z')
+    ax.tick_params(axis='z', which='major', pad=-12)
+    ax.w_zaxis._axinfo['tick']['linewidth'] = 0
+    ax.grid(False)
+    ax.elev = 30
+
+
+plt.subplots_adjust(left=0.,
+                    bottom=-.05,
+                    right=1.03,
+                    top=1.19,
+                    wspace=-0.2,
+                    hspace=-0.45)
+
+# highlight AHE
+rect_ax = fig.add_axes([0, 0, 1, 1], facecolor='none')
+rect_ax.axis(False)
```

Comment:
```suggestion
rect_ax.set_axis_off()
```
This line is causing CI failures ;)

Reply:
Thanks!